services/helper/sessions.js

```diff
@@ -1,64 +1,703 @@
+                        session._id,
+                        mentorDetails.name,
+                        session.mentorPassword
+                    );
+
+                    await sessionData.updateOneSession({
+                        _id: session._id
+                    }, {
+                        link: moderatorMeetingLink,
+                        status: "live",
+                        startedAt: new Date()
+                    })
 
-                /**
-                 * Sessions form business logic
-                 */
+                    link = moderatorMeetingLink;
+                }
 
-            } catch(error) {
+                return resolve(common.successResponse({
+                    statusCode: httpStatusCode.ok,
+                    message: apiResponses.SESSION_START_LINK,
+                    result: {
+                        link: link
+                    }
+                }));
+
+            } catch (error) {
                 return reject(error);
             }
         })
-    } 
+    }
 
-    static update(sessionId,bodyData) {
-        return new Promise(async (resolve,reject) => {
+    static setMentorPassword(sessionId, userId) {
+        return new Promise(async (resolve, reject) => {
             try {
-                 /**
-                 * Your business logic here
-                 */
-            } catch(error) {
+
+                let hashPassword = utils.hash(sessionId + userId);
+                const result = await sessionData.updateOneSession({
+                    _id: sessionId
+                }, {
+                    mentorPassword: hashPassword
+                });
+
+                return resolve(result);
+
+            } catch (error) {
                 return reject(error);
             }
         })
-    } 
+    }
 
-    static details(sessionId) {
-        return new Promise(async (resolve,reject) => {
+    static setMenteePassword(sessionId, createdAt) {
+        return new Promise(async (resolve, reject) => {
             try {
-                 /**
-                 * Your business logic here
-                 */
-            } catch(error) {
+
+                let hashPassword = utils.hash(sessionId + createdAt);
+                const result = await sessionData.updateOneSession({
+                    _id: sessionId
+                }, {
+                    menteePassword: hashPassword
+                });
+
+                return resolve(result);
+
+            } catch (error) {
                 return reject(error);
             }
         })
-    } 
+    }
 
-    static enroll(sessionId) {
-        return new Promise(async (resolve,reject) => {
+    static completed(sessionId) {
+        return new Promise(async (resolve, reject) => {
             try {
-                 /**
-                 * Your business logic here
-                 */
-            } catch(error) {
+
+                const recordingInfo = await bigBlueButton.getRecordings(sessionId);
```

Comment:
@aman-tunerlabs Since recording is not needed feature for now so we can comment this code.

Reply:
@aman-g202  Let's keep it as its a needed feature.